aten/src/ATen/core/ATenGeneral.h

```diff
@@ -4,3 +4,5 @@
 
 // TODO: Merge the *_API macros.
 #define AT_API AT_CORE_API
+#define AT_EXPORT AT_CORE_EXPORT
```

Comment:
Just FYI, if you're going to merge everything in the end, I'd rather it be called `AT_API` than `AT_CORE_API`